app/js/views/themes.jsx

```diff
@@ -0,0 +1,94 @@
+/* global React */
+
+import NavigationMenu from 'js/views/navigation-menu';
+
+export default class Themes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      themes: [
+        {
+          name: 'When no presence detected, turn off the lights',
+          conditionServiceId: 123,
+          conditionProp: true,
+          actionServiceId: 456,
+          actionProp: true,
+          enabled: true
+        },
+        {
+          name: 'When a presence is detected, take a picture',
+          conditionServiceId: 123,
+          conditionProp: true,
+          actionServiceId: 456,
+          actionProp: true,
+          enabled: false
+        }
+      ]
+    };
+
+    this.foxbox = props.foxbox;
+  }
+
+  /**
+   * Activate or deactivate a recipe.
+   *
+   * @param {number} id
+   * @param {SyntheticEvent} evt
+   */
+  handleOnChange(id, evt) {
+    const value = evt.target.checked;
+
+    this.state.themes[id].enabled = value;
+    this.setState({ themes: this.state.themes });
+  }
+
+  /**
+   * Delete a recipe.
+   *
+   * @param {number} id
+   */
+  handleOnDelete(id) {
+    this.state.themes.splice(id, 1);
+    this.setState({ themes: this.state.themes });
+  }
+
+  render() {
+    const themeItems = this.state.themes.map((theme, id) => {
+      let itemClassName = 'themes-list__item';
+      if (!theme.enabled) {
+        itemClassName += ' themes-list__item--deactivated';
+      }
+
+      return (
+        <li key={id}
+            className={itemClassName}>
+          <input className="themes-list__toggle" type="checkbox"
+                 checked={theme.enabled}
+                 onChange={this.handleOnChange.bind(this, id)}/>
+          <span className="themes-list__name">{theme.name}</span>
+          <img className="themes-list__remove" src="css/icons/clear.svg"
```

Comment:
__nit:__ can we use ```button``` here? it looks more natural here and better from the accessibility stand point.